internal/tjson/tjson.go

```diff
@@ -66,12 +66,16 @@ type tjsontype interface {
 func checkConsumed(dec *json.Decoder, r *bytes.Reader) error {
 	if dr := dec.Buffered().(*bytes.Reader); dr.Len() != 0 {
 		b, _ := io.ReadAll(dr)
-		return lazyerrors.Errorf("%d bytes remains in the decoded: %s", dr.Len(), b)
+
+		b = bytes.TrimSpace(b)
```

Comment:
Let's add a comment explaining why. We don't do that in pjson, for example.

Reply:
btw, the explanation in the PR description is great, but it is not visible when someone reads this code after the PR is merged

Reply:
fixed